src/Twitter.jl

```diff
@@ -1,3 +1,6 @@
+using CSV
+using DataFrames
```

Comment:
Why have these moved?

Reply:
These are used when `Twitter.jl` is being used, so I thought to keep them in `Twitter.jl`. Should I move them back to `CorpusLoaders` ?

Reply:
Yes all imports in `src` should be in the main file.
it makes it easier to keep track of what things are in scope, and what packages you depend on.
(I tried the other way and it eventually becomes a mess)